beginner_source/maskedtensor_overview.rst

```diff
@@ -0,0 +1,244 @@
+MaskedTensor Overview
+=====================
+
+This tutorial is designed to serve as a starting point for using MaskedTensors
+and discuss its masking semantics.
+
+Using MaskedTensor
+++++++++++++++++++
+
+Construction
+------------
+
+There are a few different ways to construct a MaskedTensor:
+
+* The first way is to directly invoke the MaskedTensor class
+* The second (and our recommended way) is to use :func:`masked.masked_tensor` and :func:`masked.as_masked_tensor` factory functions,
+  which are analogous to :func:`torch.tensor` and :func:`torch.as_tensor`
+
+  .. autosummary::
+    :toctree: generated
+    :nosignatures:
+
+    masked.masked_tensor
+    masked.as_masked_tensor
+
+Accessing the data and mask
+---------------------------
+
+The underlying fields in a MaskedTensor can be accessed through:
+
+* the :meth:`MaskedTensor.get_data` function
+* the :meth:`MaskedTensor.get_mask` function. Recall that ``True`` indicates "specified" or "valid" while ``False`` indicates
+  "unspecified" or "invalid".
+
+In general, the underlying data that is returned may not be valid in the unspecified entries, so we recommend that
+when users require a Tensor without any masked entries, that they use :meth:`MaskedTensor.to_tensor` (as shown above) to
+return a Tensor with filled values.
+
+Indexing and slicing
+--------------------
+
+:class:`MaskedTensor` is a Tensor subclass, which means that it inherits the same semantics for indexing and slicing
+as :class:`torch.Tensor`. Below are some examples of common indexing and slicing patterns:
+
+    >>> data = torch.arange(60).reshape(3, 4, 5)
```

Comment:
can you copy the output of this call and the mask call below as well? It helps the user see the original data before masking.

Reply:
Done!